pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -147,9 +148,11 @@ type VSphereConfig struct {
 
 	// Endpoint used to create volumes
 	Workspace struct {
-		VCenterIP  string `gcfg:"server"`
-		Datacenter string `gcfg:"datacenter"`
-		Folder     string `gcfg:"folder"`
+		VCenterIP        string `gcfg:"server"`
+		Datacenter       string `gcfg:"datacenter"`
+		Folder           string `gcfg:"folder"`
+		DefaultDatastore string `gcfg:"default-datastore"`
+		ComputePath      string `gcfg:"compute-path"`
```

Comment:
What is compute-path?
I don't get any idea by name?
What your should provide as value?

Reply:
This is the path to the resource pool where we create dummy VMs. Please see VSphere.setVMOptions onw how this is used. Let me know if you have a better name for this parameter.

Reply:
I got the use of this field. But customer will not know about the use just at lookig at the name "compute-path". how about VMBasePath?

Reply:
VMBasePath sounds like a directory path to me. We want to be able to convey that this path is for the cluster/host/resource pool within the datacenter. Any better names? How about resourcePoolPath or computeResourcePath?

Reply:
ResourcePoolPath looks little more intuitive

Reply:
I'll update the config parameter. Do you have any other review comments on this PR?